src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -213,32 +214,41 @@ class SystemDictMonitor : public CHeapObj<mtWisp> {
     Monitor*  _monitor;
   public:
     SystemDictMonitor(Monitor* monitor): _monitor(monitor) { }
-    virtual void lock(BasicLock* lock, TRAPS)    { _monitor->lock();   }
-    virtual void unlock(BasicLock* lock, TRAPS)  { _monitor->unlock(); }
-    virtual void wait(BasicLock* lock, TRAPS)    { _monitor->wait();   }
-    virtual void notify_all(TRAPS)               { _monitor->notify_all(); }
-    virtual void set_obj_lock(oop obj, TRAPS) { ShouldNotReachHere(); }
+    virtual void lock(BasicLock* lock, Thread* current)    {
```

Comment:
use the same coding style

Reply:
Sorry but I didn't follow you here. Isn't this line the same coding style already?